install/local/install-scrypted-dependencies-win.ps1

```diff
@@ -34,7 +34,8 @@ $SCRYPTED_HOME_ESCAPED_PATH = $SCRYPTED_HOME.replace('\', '\\')
 npm install --prefix $SCRYPTED_HOME @koush/node-windows --save
 
 $NPX_PATH = (Get-Command npx).Path
-$NPX_PATH_ESCAPED = $NPX_PATH.replace('\', '\\')
+# The path needs double quotes to handle spaces in the directory path
+$NPX_PATH_ESCAPED = `"${$NPX_PATH.replace('\', '\\')}"`
```

Comment:
@koush I couldn't quite figure out why you changed the NPX path to be the full path in https://github.com/koush/scrypted/commit/ddb6a3a47d68503090dd781522303a87869575a1 as opposed to just `npx.cmd`, but I guess you had a good reason.

Reply:
I'm guessing because it's running as a service it needs the full path since it doesn't have the npm directory in the `Path` environment variable?